con-master/src/main/java/ba/codecentric/master/controller/FacilityController.java

```diff
@@ -0,0 +1,39 @@
+package ba.codecentric.master.controller;
+
+import ba.codecentric.base.domain.Facility;
+import ba.codecentric.base.service.FacilityService;
+import ba.codecentric.base.service.ImageService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.Valid;
+import java.io.IOException;
+
+@RestController
+public class FacilityController {
+
+    private final FacilityService facilityService;
+    private final ImageService imageService;
+
+    @Autowired
+    public FacilityController(FacilityService facilityService, ImageService imageService) {
+        this.facilityService = facilityService;
+        this.imageService = imageService;
+    }
+
+    @RequestMapping(value = "/facility", method = RequestMethod.POST, consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```

Comment:
Are you naming you URL's in plural or singular? So far it was in plural. Therefore, `"/facilities"`